P5/Source/Specs/glyphName.xml

```diff
@@ -27,6 +28,15 @@ $Id$
   <content>
     <textNode/>
   </content>
+  <constraintSpec scheme="schematron" validUntil="2021-07-09"
```

Comment:
Delete the entire `<constraintSpec>`. The deprecation Schematron check will (or at least should) be automatically generated at build time because of the `@validUntil` on the `<elementSpec>`.